lib/App/DuckPAN/DDG.pm

```diff
@@ -77,8 +77,7 @@ sub get_blocks_from_current_dir {
 		else {
 			$failed_to_load{$arg} =
 				'Could not retrieve metadata - please ensure the Instant Answer page ' .
-				'is in development status or later.';
-			next;
+				'is in development status or later; the IA has still been loaded.';
```

Comment:
How about?

1. Use `emit_notice` to warn that it's not in metadata
2. Set `$class = $arg` which will fail if using an ID.